pkg/gadget-service/api/api.proto

```diff
@@ -50,6 +50,29 @@ message GadgetRunRequest {
   int64 timeout = 13;
 }
 
+message OCIGadgetRunRequest {
```

Comment:
It seems used client-side but I don't see the implementation server-side. Is it implemented?

Reply:
It's used [here](https://github.com/inspektor-gadget/inspektor-gadget/pull/2512/files#diff-2fc6ea945ba613e34da8f66d32df8a5270ce422ccd43a855a7d3c1c10588a9aeR140) 😄